### DISCUSSION TITLE
Improve positioning of tabs on metrics.lecoq.io

### DISCUSSION THREAD

**Discussion by Andre601** [lowlighter/metrics#900] [General]
When watching https://metrics.lecoq.io do the tabs for "Metrics Preview", "Action Code", "Markdown Code" and "Metrics Insight" look quite a bit off...

They are fairly far to the left, which makes them feel somewhat out of place.
I suggest to make them align with the "box" that contains the example readme, so that "Metrics Preview"'s left side is aligned with the box's left side.

Screenshot of the visual issue:
![image](https://user-images.githubusercontent.com/11576465/154768134-cca7ba86-7b33-4cf6-9f25-9ea91f4422e6.png)

**Comment by DalpatRathore** [comment]
Hi! @Andre601,
&nbsp;&nbsp;&nbsp;&nbsp; I think you want to align **navbar tabs** and **metrics preview** as shown in screenshot:

![capp](https://user-images.githubusercontent.com/69510006/154799312-d2a1bfdd-01f9-412b-b1ec-7de659803006.png)

**Reply by Andre601** [reply]
Yes.

**Reply by DalpatRathore** [reply]
I think it is doable. I have used the Google Chrome Devtools to showcase screenshot results. Let's wait and see the metrics maintainer's response.

**Reply by lowlighter** [reply]
Yeah it's not a stylistic choice 😅 
I think some time in the past the sidebar width was increased, but tabs didn't get updated

If you already know how to fix it, you can submit a pull request for it 👍

**Reply by DalpatRathore** [reply]
I am good at Frontend - CSS, JS, and React but the web app is written in Vue. Though the stylistic issue can be easily fixed:
- Moving navbar tabs into metrics preview box. 
- Using media queries to re-arrange.

**Reply by DalpatRathore** [reply]
I had made pull request #902 regarding this issue. Though I have used Google Chrome Devtools to solve it and then pasted code in the relevant file. Let's see if it works and solves the misplaced tabs menu issue.

**Reply by DalpatRathore** [reply]
Pull requests #903
